### ISSUE TITLE
pod 'Mixpanel/AppExtension' fails on Archive

### ISSUE THREAD

**Issue by kevboh** [mixpanel/mixpanel-iphone#266]
I have an app with a Podfile that looks roughly like this:

```
platform :ios, '8.0'

use_frameworks!

target 'MyApp' do
	pod 'Mixpanel'
end

target 'MyAppShareExtension' do
	pod 'Mixpanel/AppExtension'
end

post_install do |installer|
  installer.project.targets.each do |target|
    target.build_configurations.each do |config|
      preprocessor_definitions = config.build_settings['GCC_PREPROCESSOR_DEFINITIONS']
      preprocessor_definitions = ['$(inherited)'] if preprocessor_definitions == nil
      preprocessor_definitions.push 'MIXPANEL_NO_IFA' if target.to_s.include? 'Mixpanel'
      config.build_settings['GCC_PREPROCESSOR_DEFINITIONS'] = preprocessor_definitions
    end
  end
end
```

In my share extension I import Mixpanel like this:

`import Mixpanel`

Building works fine, but on Archive I get the following error (and no archived app):

`/path/to/app/<module-includes>:1:1: Umbrella header for module 'Mixpanel' does not include header 'Pods-MyAppShareExtension-Mixpanel-umbrella.h'`

Any ideas? Could the app target's Mixpanel framework be linked instead?

**Comment by samgreen** [comment]
This should be resolved in v2.8.2, can you try to do a `pod update Mixpanel` and let me know how that goes?

**Comment by samgreen** [comment]
Closing this issue. Please open another issue if you're still having trouble.
